utils/fixed_decimal/Cargo.toml

```diff
@@ -45,6 +45,7 @@ path = "src/lib.rs"
 
 [features]
 default = []
+std = ["displaydoc/std"]
```

Comment:
Question: what does `"displaydoc/std"` do and why do you need it?

Reply:
We don't actually need it, I can remove this bit.